code/include/swoc/Errata.h

```diff
@@ -186,36 +210,100 @@ class Errata {
   template <typename... Args> Errata(Severity severity, std::string_view fmt, Args &&... args);
   template <typename... Args> explicit Errata(std::string_view fmt, Args &&... args);
 
-  /** Add a new message to the top of stack with severity @a level and @a text.
+  /** Add an @c Annotation to the top with @a text.
    * @param text Text of the message.
    * @return *this
    *
-   * The error code and severity are set to defaults.
+   * The error code is set to the default.
+   * @a text is localized to @a this and does not need to be persistent.
    */
   self_type &note(std::string_view text);
 
-  /** Add a new message to the top of stack with severity @a level and @a text.
+  /** Add an @c Annotation to the top with @a text and local @a severity.
+   * @param severity The local severity.
+   * @param text Text of the message.
+   * @return *this
+   *
+   * The error code is set to the default.
+   * @a text is localized to @a this and does not need to be persistent.
+   */
+  self_type &note(Severity severity, std::string_view text);
+
```

Comment:
Thanks! I'm in the process making proxy-verifier changes for this.

We should add this note(Severity) interface to Rv as well.

Reply:
Thanks for adding this. I verified Rv::note(Severity) works in proxy verifier.